core/src/main/scala/org/bitcoins/core/protocol/script/ScriptPubKey.scala

```diff
@@ -866,6 +866,70 @@ object NonStandardNotIfConditionalScriptPubKey
   }
 }
 
+sealed trait P2PKWithTimeoutScriptPubKey extends RawScriptPubKey {
+
+  val pubKey: ECPublicKey =
```

Comment:
can you make these all `lazy`, i worry about weird initialization errors if they aren't `lazy`